src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingWorkCoordinator.cs

```diff
@@ -31,7 +31,7 @@ internal sealed partial class UnitTestingWorkCoordinator : IUnitTestingWorkCoord
 
             private readonly CountLogAggregator<WorkspaceChangeKind> _logAggregator = new();
             private readonly IAsynchronousOperationListener _listener;
-            private readonly IWorkspaceConfigurationService? _workspaceConfigurationService;
+            private readonly Microsoft.CodeAnalysis.SolutionCrawler.ISolutionCrawlerOptionsService? _solutionCrawlerOptionsService;
```

Comment:
Note the unit testing fork of the solution crawler was still using the same underlying option as the mainline solution crawler; this maintains that.